### DISCUSSION TITLE
frpc v0.51.0-sakura-2奇怪问题

### DISCUSSION THREAD

**Discussion by eya46** [natfrp/wiki#413] [使用问题]
使用命令-f token:1,2,3启动，只要启动隧道数量为三个就报错：`2024/01/18 19:53:15 [W] [请更新 frpc 到 v0] 登录节点失败: dial tcp: lookup 请更新 frpc 到 v0.42.0-sakura-6: no such host, 请检查网络连接`
（TCP,WOL,HTTP，每次都是WOL没法启动，换了位置也不行）
试过win10 amd64、linux arm64，都报这错
缩短到两个正常，更多的没有尝试
![image](https://github.com/natfrp/wiki/assets/61458340/fcbd73c8-bd7a-4b5b-a9e9-69d2393bb4f5)

**Comment by fakeboboliu** [comment]
感谢您提供的信息，我们已经确定并正在修复多连接模式时 WOL 隧道的配置解析问题，

目前建议您即使没有此问题总是为 WOL 隧道单独开启一个进程，
这样也能保证 WOL 隧道的可用性，防止误 kill 等情况发生。
